assets/helpers/externalLinks.js

```diff
@@ -52,7 +52,7 @@ const defaultPromos: PromoCodes = {
   PaperAndDigital: getPromoCode('PaperAndDigital', 'GXX83X'),
 };
 
-const customPromos : {
+const customPromos: {
```

Comment:
Worth adding [prettier and the eslint plugin](https://github.com/prettier/prettier) to automate fixing these small formatting issues?